charts/gardener-dashboard/templates/configmap.yaml

```
@@ -300,4 +300,17 @@ data:
 {{ toYaml . | trim | indent 12 }}
           {{- end }}
         {{- end }}
+      {{- end }}
+      {{- if .Values.frontendConfig.vendorHints }}
+      vendorHints:
+        {{- range .Values.frontendConfig.vendorHints }}
+        - vendorNames:
+            {{- range .vendorNames }}
+            - {{ . }}
+            {{- end }}
+          hintMessage: {{ quote .hintMessage }}
+          {{- if .hintType }}
+          hintType: {{ .hintType }}
+          {{- end }}
+        {{- end }}
```

The property is already called `vendorHints`. It would not repeat vendor and hint in child properties. Type from my point of view should be named severity. 
```suggestion
        {{- range .Values.frontendConfig.vendorHints }}
        - matchNames: {{ toJson .matchNames }}
           message: {{ quote .message }}
           {{- if .severity }}
           severity: {{ .severity }}
           {{- end }}
        {{- end }}
```

---

changed as suggested